sys-fs/genfstab/genfstab-28-r1.ebuild

```diff
@@ -22,3 +24,9 @@ BDEPEND="
 src_test() {
 	emake check
 }
+
+src_install() {
+	dobin ${PN}
+	doman doc/${PN}.8
+	newbashcomp "completion/${PN}.bash" "${PN}"
```

Comment:
Using PN in src_install is *bad* in case the package gets renamed to arch-genfstab or whatever in the future. Doesn't really add value unlike doing it in SRC_URI etc.